### ISSUE TITLE
No Option To Execute Code

### ISSUE THREAD

**Issue by ColeWunderlich** [giuspen/cherrytree#488]
On version 0.37.6  there is no option to run code blocks.  It does not appear in the file menu or in the right click menu on a code block.

This is being run on Ubuntu 18.04

**Comment by EllieBorden** [comment]
That feature was implemented in version 0.38.

Follow the steps in this link to install the updated version:
http://giuspen.com/cherrytreemanual/#_ubuntu_debian

**Comment by ColeWunderlich** [comment]
Ah, sorry I didn't realize it was that recent.  Newest version installed and code execution works fine.

Two quick remarks:
1. It looks like 0.37.6 is the latest version in the standard repository, might be worth updating if 0.38 is stable.

2. It might be worth adding install instructions for users without SU privileges.  I was able to install it by simply extracting the .deb (using `dpkg -x`) and then linking the executable to /home/bin (my directory for local installs)
```
dpkg -x cherrytree_0.38.8-0_all.deb /path/to/desired/dir

ln -s /path/to/desired/dir/cherrytree ~/bin/cherrytree
```
